mirage/config.js

```diff
@@ -214,8 +218,25 @@ export default function configure() {
     let params = req.queryParams;
     let type = params['filter[type]'];
     let selected = params['filter[selected]'];
+    let name = params['filter[name]'];
+    let isxn = params['filter[isxn]'];
+    let subject = params['filter[subject]'];
+    let publisher = params['filter[publisher]'];
+    let queryFiltered = false;
```

Comment:
We should default this to `true` and remove the last `else` below. This keeps it consistent with `filtered` and says that we should return true unless a more specific filter is provided.

Reply:
Thanks for all the comments @wwilsman - will refine as suggested. As an added note - reviewed functionality with Khalilah (when switching between SearchOptions) and we will be maintaining the search term rather than clearing it out